.github/workflows/check-for-tutorial-prs.yml

```
@@ -0,0 +1,34 @@
+# This workflow detects PRs that make changes to lms/templates/dashboard.html
+# and only lms/templates/dashboard.html. This is the file that users are
+# guided through changing in the Open edX tutorial:
+# https://docs.openedx.org/en/latest/developers/quickstarts/first_openedx_pr.html#exercise-update-the-learner-dashboard
+
+# If this is the only file changed in the PR, we comment on the PR congratulating
+# the user and letting others know that this is not a community PR in need of
+# review. CODEOWNERS will tag a triaging team to provide reviews & ultimately
+# close the PR.
+
+name: Welcome contributors making their first PR from the tutorial
```

The "name" field usually displays in a pretty small box, so you might want to make this the description and have an abbreviated version for the name.
```suggestion
name: Contributor Tutorial Welcome
description: Welcome contributors making their first PR from the tutorial
```